### PR TITLE
DEV-1176 update example to work with latest lib version

### PR DESCRIPTION
**Pull request by yovasx2**
A few things were addressed in this PR:
- Bump up node-oidc-client lib to the latest version (3.0.6)
- Update the example to implement client_secret instead of the outdated private_key_jwt
- Remove jwks form the client_secret implementation example to make it simpler

https://identity-online.atlassian.net/browse/DEV-1176

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,43 +1,33 @@
...
-}();
-
 const router = express.Router();
 
-router.get('/', async (req, res, next) => {
-  res.render('index', { title: 'RP Example' });
+const idPartner = new IDPartner({
+  client_id: 'CHANGE_ME_CLIENT_ID',
+  client_secret: 'CHANGE_ME_CLIENT_SECRET',
+  callback: 'http://localhost:8000/auth/callback',
```
Comment (francisco-idpartner) [inline routes/index.js:9]: should this be ngrok?

Comment (yovasx2) [reply routes/index.js:9]: not necessary, it works without and we make it simple, i include it in the further steps if u want to customize other things

Comment (francisco-idpartner) [reply routes/index.js:9]: Discussed offline - the reason for not using ngrok is that this URL is consumed via redirect. It's not consumed via direct request (as it used to due to JWKS)

**Review by francisco-idpartner — APPROVED** [review]
LGTM

**Pull request merged**